seisidd/devices/detectors.py

```diff
@@ -6,43 +6,66 @@
 NOTE:
 * Retiga area detector is not readily available in Ophyd
 * Retiga area detector camera needs to be either setup here or PR to ophyd 
+* Changed to PointGrey Detectors in this example
```

Comment:
Perhaps we should make a request in Ophyd to ask for support of Retiga detector.

Reply:
Currently we are closer to the FLIR/PG option, but this won't hurt.